addons/hr_skills/static/src/fields/resume_one2many/resume_one2many.xml

```diff
@@ -40,7 +40,7 @@
             <td class="o_data_cell pt-0" t-on-click="(ev) => this.onCellClicked(record, null, ev)">
```

Comment:
Commit message:

> We do such "display" correction to avoid multiple upgrades, and we will do the
right correction in 16.3.

Almost, isn't it "17.3"? ^^
You can say "master" actually to avoid forcing a timeline on the subsequent fix, even if it will most likely land in 17.3 indeed.